### PR TITLE
v3.0 Release

### PR DESCRIPTION
**Pull request by j-d-b**
# WZDx Version 3.0 Release

## Features

* Restrict `version` format to `major.minor` and enforce via v3.0 JSON schema
* Add `order` property to **lane** entity to indicate a lane's position in sequence on the roadway
* Remove `lane_edge_reference` and standardize on counting lanes from the *left* side of the roadway
* Add `event_type` property to the **road event** entity and new **Event Type Enumerated Type** with the values `work-zone` and `detour`, to support adding detour information (and more in the future)
* Add **relationship** entity (one per **road event** entity) to enable describing relationships (hierarchical and sequential) between road events and other non-WZDx entities (such as a work zone phase or project)
* Add **road_event_data_source** entity to allow specifying multiple data sources within a single feed
* Streamline **metadata** (previously given in an external file, by URL) and add relevant fields to the **road_event_feed_info** and **road_event_data_source** entities

## Fixes

* Require `restriction_type` if providing a lane-level restriction
* Change **Spatial Verification Enumerated Type** and **Time Verification Enumerated Type** values to be all lowercase
* Remove all plural or nonsensical lane types from the **Lane Type Enumerated Type** and clarify that each WZDx lane should represent a *single* lane on the roadway

### CODE REVIEW

File: spec-content/data-tables/road_event_feed_info.md [hunk 1]
```diff
@@ -1,12 +1,22 @@
...
-version |	Text |	The specification version used to create the data feed |	Optional	 |
+**feed_info_id** |	ID |	Unique identifier for the data feed. | Required | Primary Key Recommendations on the format of this unique identifier will be made in the future. | 
+**publisher** | Text | The organization responsible for publishing the feed. | Required | Example: State DOT
+**version** | Text | The specification version used to create the data feed in `major.minor` format. | Required | Examples: `1.1`, `2.0`
+**update_date** |	DateTime | The UTC date and time when the data feed was last updated. |	Required | All date/time formats shall use ISO 8601 Data elements and interchange formats – Information interchange. Example: `2016-11-03T19:37:00Z`
+**update_frequency** | Integer | The frequency in seconds at which the data feed is updated | Optional | Example: 60
+**contact_name** | Text | The name of the individual responsible for the data feed | Optional | Example: Jo Help
+**contact_email** | Text  | The email address of the individual responsible for the data feed | Optional | Example: abc@testcity1.gov
```
Comment (chuehlien) [inline spec-content/data-tables/road_event_feed_info.md]: Do we want to rephrase the `contact_name` and `contact_email` descriptions in the `road_event_feed_info` and `road_event_data_source` to allow listing of an team name/email?

e.g. "The name of the individual or group responsible for the data feed" and "The email address of the individual or group responsible for the data feed"

**Review by chuehlien — APPROVED** [review]

**Review by Mahsa-Ettefagh — APPROVED** [review]

File: spec-content/data-tables/road_event_feed_info.md (hunk 1 above)
Comment (j-d-b) [reply spec-content/data-tables/road_event_feed_info.md]: Yes I like that.

Comment (j-d-b) [reply spec-content/data-tables/road_event_feed_info.md]: Done

**Comment by j-d-b** [comment]
@Mahsa-Ettefagh @chuehlien is there anything remaining/anyone else reviewing before this can be merged?

File: create-feed/README.md [hunk 2]
```diff
@@ -41,32 +43,33 @@ The following WZDx feed examples include all optional fields. An example file is
...
+- **Discussing and Reviewing Commits** - This part of the process allows reviewers on your team or the DAVI Data Team check that the spec is properly implemented (e.g., that data types are correctly specified in the JSON schema).
+- **Merging to Master Branch** - Once your commits have been reviewed and approved, you can push your changes to merge with the master branch.
 
 A more in-depth [tutorial is available here](https://guides.github.com/introduction/flow/).
 
 To start collaborating via GitHub:
-- [Join GitHub](https://github.com/) by creating a free [organization account and user accounts](https://help.github.com/articles/differences-between-user-and-organization-accounts/) for your team.
-- If you are a data producer contact [Nate Deshmukh-Towery](nate.deshmukh-towery@dot.gov) to request your own sub-repository to host your project.
+- [Join GitHub](https://github.com/) by creating free [organization and user accounts](https://help.github.com/articles/differences-between-user-and-organization-accounts/) for your team.
+- If you are a data producer, contact [Nate Deshmukh-Towery](nate.deshmukh-towery@dot.gov) to request your own sub-repository to host your project.
```
Comment (chuehlien) [inline create-feed/README.md:70]: `...to request your own sub-repository to host your project.`

@j-d-b @Mahsa-Ettefagh @DeraldDudley Is this still accurate?

**Review by j-d-b — COMMENTED** [review]
Another good catch, I need to be sure to be comprehensive

File: create-feed/README.md (hunk 2 above)
Comment (j-d-b) [reply create-feed/README.md:70]: I have no idea, @natedeshmukhtowery ?

Comment (natedeshmukhtowery) [reply create-feed/README.md:70]: We have not yet done this for any data producer, and it doesn't seem relevant. I would suggest removing it.

**Comment by Mahsa-Ettefagh** [comment]
@j-d-b all reviews are completed and this PR can be merged

File: create-feed/README.md (hunk 2 above)
Comment (j-d-b) [reply create-feed/README.md:70]: Completed in https://github.com/usdot-jpo-ode/jpo-wzdx/pull/132

**Pull request merged**